src/index.ts

```diff
@@ -1799,29 +1800,20 @@ async function claimLinkGasless({
 	link,
 	recipientAddress,
 	APIKey,
-	baseUrl = 'https://api.peanut.to/claim',
+	baseUrl = 'https://api.peanut.to/claim-v2',
```

Comment:
Current deployment of the api is on https://peanut-api-ts-9lo6.onrender.com -> pls update baseUrl :)

Reply:
Why? The default value here is intended for production use and the current deployment is there only temporarily

Reply:
We'll probably have the ts-api in staging environment first. Till it's had enough QA in staging, we'll move to prod

Reply:
We can discuss this with Hugo then, as I disagree